rllib/utils/replay_buffers/prioritized_episode_replay_buffer.py

```diff
@@ -536,7 +532,7 @@ def sample(
         if include_infos:
             ret.update(
                 {
-                    SampleBatch.INFOS: infos,
+                    Columns.INFOS: infos,
```

Comment:
Nice catch! :)

Reply:
Yup that made it somehow over this long time in there :D